app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsController.scala

```diff
@@ -80,12 +80,30 @@ class ReliefDetailsController @Inject() (
     ) => Future[Result]
   ): Future[Result] =
     request.sessionData.flatMap(s => s.journeyStatus.map(s -> _)) match {
+      case Some(
+            (s, r @ FillingOutReturn(_, _, _, d: DraftSingleDisposalReturn, _))
+          ) if d.triageAnswers.isPeriodOfAdmin() =>
+        d.reliefDetailsAnswers.fold[Future[Result]](
+          f(s, r, d, IncompleteReliefDetailsAnswers.empty)
+        )(relief =>
+          f(
+            s,
+            r,
+            d,
+            relief.fold(
+              _.copy(lettingsRelief = Some(AmountInPence.zero)),
```

Comment:
if you do a global search for `unset(_.lettingsRelief)` you'll see that in a couple of places` the `lettingsRelief` field is being unset when certain answers change. This needs to not happen when it's period of admin, otherwise he work to pre-populate the field will be undone when those answers change.

Reply:
this happens in a number of controllers - you'll have to do a global search on the whole project to get all the instances

Reply:
oops, I didn't see this word "global" search. Will fix it